content/updates/2020-07/1-rules-UI.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/rules_UI.png
+pim_announcement_alt_img: "The UI to manage the rules"
+pim_announcement_audience:
+- EE
+---
+
+# A brand new UI to manage your rules
+
+In order to let everyone use the rules engine and boost your productivity, we have created a user interface to manage your rules directly in the PIM.  
+
+We know that your time is precious and that you always need to work efficiently. That's why we built the rules engine a few years ago.
+Indeed, this feature allows you to automate actions on a selection of products/product models. But until now, the only way to create/update rules was via a YML file import, which was not simple enough for our non-technical users.  
```

Comment:
```suggestion
```

Reply:
I wouldn't insist on the fact that the YML files are not easy to use for non tech users.